### ISSUE TITLE
Insertion Point Unintentionally Moves to Line 1

### ISSUE THREAD

**Issue by boredbum** [FreeCodeCamp/FreeCodeCamp#10642]
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36</code>.

While typing code, insertion point will sometimes move to Line 1.  The problem is difficult to recreate, but it's happened so far when I've inputted backspace, ctrl+backspace, home + backspace, shift + 7 (&).  This problem has occurred across multiple challenges and only after the implementation of the new site.  I've tried controlling for user error by locking my touchpad, problem still persists with occasional frequency.

```js

**Comment by BerkeleyTrue** [comment]
Closing as duplicate of the codemirror bug

Please see http://forum.freecodecamp.com/t/creating-a-new-github-issue/18392

Happy Coding!
